python/lsst/ip/isr/isrTask.py

```diff
@@ -1203,7 +1204,7 @@ def run(self, ccdExposure, camera=None, bias=None, linearizer=None, crosstalkSou
                 if self.config.doOverscan and not badAmp:
                     # Overscan correction on amp-by-amp basis.
                     overscanResults = self.overscanCorrection(ccdExposure, amp)
-                    self.log.debug(f"Corrected overscan for amplifier {amp.getName()}.")
+                    self.log.debug("Corrected overscan for amplifier %s." % (amp.getName()))
```

Comment:
This should be:

    self.log.debug("Corrected overscan for amplifier %s.", amp.getName())

This way, the stringification only occurs if the log threshold is sufficient, rather than being done always.

Here and for all other changes involving `self.log.whatever`.